src/components/ClickableStyle/ClickableStyle.module.css

```diff
@@ -22,6 +22,7 @@
   border-style: solid;
   border-radius: var(--eds-theme-border-radius);
   cursor: pointer;
+  text-decoration: none;
```

Comment:
I added this line because the link stories were showing underlines in the button variants. (This didn't come up before because none of the stories had an `href` prop, which is what activates the link styles.)

Reply:
You got it 👍